server/src/main/java/keywhiz/service/resources/StatusResource.java

```diff
@@ -1,49 +1,42 @@
 package keywhiz.service.resources;
 
-import com.codahale.metrics.annotation.Timed;
 import com.codahale.metrics.annotation.ExceptionMetered;
+import com.codahale.metrics.annotation.Timed;
 import com.codahale.metrics.health.HealthCheck;
 import com.codahale.metrics.health.HealthCheckRegistry;
 import io.dropwizard.setup.Environment;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
 import java.util.SortedMap;
 import javax.inject.Inject;
 import javax.ws.rs.GET;
-import javax.ws.rs.InternalServerErrorException;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import static java.util.stream.Collectors.toList;
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 
 /** Serve status information */
 @Path("/_status")
 @Produces(APPLICATION_JSON)
 public class StatusResource {
   private Environment environment;
+  private static final Logger logger = LoggerFactory.getLogger(SecretDeliveryResource.class);
 
   @Inject public StatusResource(Environment environment) {
     this.environment = environment;
   }
 
   @Timed @ExceptionMetered
   @GET
-  public boolean get() {
+  public Response get() {
     HealthCheckRegistry checks = this.environment.healthChecks();
     SortedMap<String, HealthCheck.Result> results = checks.runHealthChecks();
 
-    List<String> failing = results.entrySet().stream()
-        .filter(r -> !r.getValue().isHealthy())
-        .map(Map.Entry::getKey)
-        .collect(toList());
-
-    if (!failing.isEmpty()) {
-      throw new InternalServerErrorException(
-          "failing health checks: " + Arrays.toString(failing.toArray()));
+    if (results.entrySet().stream().anyMatch(r -> !r.getValue().isHealthy())) {
+      logger.warn("Health checks failed: {}", results);
```

Comment:
The failing health checks should warn by themselves, we don't want to depend on someone hitting /_status to get warn messages.